src/wx/main_toolbar.cpp

```diff
@@ -107,6 +107,8 @@ class WXMainToolbar : public MainToolbar
             m_tb->SetToolDisabledBitmap(m_idUpdate, wxArtProvider::GetBitmap("poedit-update@disabled", wxART_TOOLBAR));
             #endif
         }
+        // Without below line tooltip is not updated, at least on Linux
+        m_tb->SetToolShortHelp(m_idUpdate, tool->GetShortHelp());
```

Comment:
Done as separate commit 33bd05108334c35bc7d3e0c1c0192e91b92f4751 but allow myself kindly disagree on that explanation of it should be removed from comment. Since it's* Workaround of wxWidgets issue

Reply:
You do have a point.

I guess my instinctive reaction to why I saw this comment as code smell was the uncertainty of it. Such vague comments tend to be cargo-culted and remain long after the underlying issue in the library was fixed and often indicate that the underlying issue isn't fully understood, which may mean the fix may not be fully correct.

Here, the underlying issue is indeed a discrepancy in wxWidgets' platform implementations — and while I have a tendency to try to fix thing in there due to my (mostly) past involvement with wx, it would seem rather impractical here.

As this is unrelated to the rest and fixes an actual bug, I pushed what I think is a more "proper" fix to master in b3a534dbf77f1c371a04a0d4036baa08ca5e399c — change the way the tooltip is set instead of setting it twice to nudge wxGTK. Links to relevant wx issues/code included in that commit.

So thanks for spotting the bug and the fix!

Reply:
I got your point and now it fully makes sense for me to avoid comments in the code in such cases. No problem that you pushed it to master outside of this PR, the main is just we make Poedit better :)